modular-docs-manual/content/topics/module_guidelines-assembly.adoc

```diff
@@ -3,6 +3,11 @@
 
 The required parts of an assembly are the introduction and modules. Optionally, an assembly can also include prerequisites and additional resources.
 
+[discrete]
+== Assembly title
+
+The title of an assembly is a gerund phrase, such as *Encrypting block devices using LUKS*.
```

Comment:
```suggestion
The title of an assembly is a gerund or noun phrase, for example:
* Encrypting block devices using LUKS
* How to Register Ceph with Satellite 6
```

Reply:
@mjahoda thanks for taking the initiative with this.

Reply:
> @mjahoda thanks for taking the initiative with this.

I'm sorry that it took so long.

To your suggestion: The fact that it can be also a noun phrase is new for me. This is another good argument for adding this definition to the guide :-). However, the second provided example use title case. Therefore, I will incorporate your suggestion manually.

Reply:
@emmurphy1 I did some research and I recalled the Write Better Titles and Headings course [1]. Ingrid's comment:

> Assembly headings should be gerunds if the assemblies contain procedures. From the templates:
> [id="assembly-my-user-story_{context}"]
> = My user story
> ////
> Be sure to include a line break between the title and the :context: variable and the :context: variable and the assembly introduction.
> 
> If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
> ////

...reminded me that we have the corresponding guidance in the templates already :-)

[1] https://360.articulate.com/review/content/1f7f2e1c-fceb-414b-83e1-59aff6a0eaf6/review